TOPdeskPS/functions/Change Management/New-TdChangeProgress.ps1

```diff
@@ -0,0 +1,75 @@
+﻿function New-TdChangeProgress {
+    <#
+.SYNOPSIS
+    creates a new action for specified change
+.DESCRIPTION
+    creates a new action for specified change
+.PARAMETER ChangeId
+    ID of the Change
+.PARAMETER MemoText
+    text to be added to memo field
+.PARAMETER Type
+    The type of this progress trail entry.
+.PARAMETER Confirm
+    If this switch is enabled, you will be prompted for confirmation before executing any operations that change state.
+.PARAMETER WhatIf
+    If this switch is enabled, no actions are performed but informational messages will be displayed that explain what would happen if the command were to run.
+.EXAMPLE
+    PS C:\> New-TdChangeProgress -ChangeId $ChangeId -MemoText 'this is a memo'
+    addes a new memo to the specified change
+#>
+    #TODO this is a duplicate of New-TdChangeAction ... neither of these work
```

Comment:
Let's test and determine if this is worht adding at this moment. also if we need to open up a direct ticket with Td